verification/curator-service/ui/src/components/LinelistTable.test.tsx

```diff
@@ -90,7 +90,12 @@ it('loads and displays cases', async () => {
 
     const { findByText, findByTestId } = render(
         <MemoryRouter>
-            <LinelistTable user={curator} />
+            <LinelistTable
+                user={curator}
+                setSearchLoading={(x: boolean): void => {
+                    console.log(`search loading ? ${x}`);
```

Comment:
I don't think these logs are useful, remove? IMO it's okay to set an empty function in tests

Reply:
done